server/routes/utils/authentication.js

```diff
@@ -0,0 +1,26 @@
+const Utility = require(`${serverRoot}/lib/Utility`);
+
+const authentication = (req, res, next) => {
+  try {
+    let authCheck = false;
+    if (typeof req.session !== 'undefined') {
```

Comment:
I have removed the optional chaining as it is only supported in node 14. See Browser Compatibility section here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining.

Reply:
Aaaw, thats a real shame.